### PR TITLE
GameLoop with GUI, refactoring into packages

### PR DESCRIPTION
**Pull request by konradpabjan**

### CODE REVIEW

**Review by anipmehta — COMMENTED** [review]
Can you change the package name to lowercase(Model to model) as mentioned in java naming convention.

File: src/main/Controller/GameLoop.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    public GameLoop(MapMakerView mapMakerView, TileSelectorView tileSelectorView){
+        this.mapMakerView = mapMakerView;
+        this.tileSelectorView = tileSelectorView;
+
+        animationTimer = new AnimationTimer()
+        {
+            public void handle(long currentNanoTime)
+            {
+                // Tell MapMakerView to render
+                mapMakerView.render();
```
Comment (gavinscheele) [inline src/main/Controller/GameLoop.java]: is this just max frame rate? or do we limit it somewhere else?

Comment (konradpabjan) [reply src/main/Controller/GameLoop.java]: Yeah, the frame rate is controlled in that loop. The animation timer actually caps it automatically at 60FPS and depending on performance will drop the frame rate if necessary. Here is an article that somewhat explains what is going on. The animation timer can be used for a whole lot of things but we're using it to keep track of frame rate. https://tbeernot.wordpress.com/2011/11/12/javafx-2-0-bubblemark/

**Comment by konradpabjan** [comment]
Fixed package names to lowercase

**Review by alindenberg — APPROVED** [review]

**Pull request merged**